src/main/java/org/apache/commons/imaging/formats/webp/WebPImagingParameters.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.imaging.formats.webp;
+
+import org.apache.commons.imaging.common.XmpImagingParameters;
+
+public class WebPImagingParameters extends XmpImagingParameters {
```

Comment:
Even small public methods and classes need a javadoc. It normally must include

- brief comment
- @since tag
- iff it's useful, tell users whether it's thread-safe or not
- if it contains complicated code, you can have a longer comment, include code examples, links to spec, etc.

Reply:
e.g. https://github.com/apache/commons-imaging/blob/cb0b52e5b82321cac1d6b4c97db7d58375322733/src/main/java/org/apache/commons/imaging/formats/png/PngImagingParameters.java#L23-L26